mysql/const.go

```diff
@@ -657,21 +653,6 @@ func FormatSQLModeStr(s string) string {
 	return strings.Join(nonEmptyParts, ",")
 }
 
-// GetSQLMode gets the sql mode for string literal. SQL_mode is a list of different modes separated by commas.
-// The input string must be formatted by 'FormatSQLModeStr'
-func GetSQLMode(s string) (SQLMode, error) {
-	strs := strings.Split(s, ",")
-	var sqlMode SQLMode
-	for i, length := 0, len(strs); i < length; i++ {
-		mode, ok := Str2SQLMode[strs[i]]
-		if !ok && strs[i] != "" {
-			return sqlMode, newInvalidModeErr(strs[i])
-		}
-		sqlMode = sqlMode | mode
-	}
-	return sqlMode, nil
-}
-
```

Comment:
This function is used in tools and should not be removed.

Reply:
thanks....I forgot tools use this package...let me take a look...

Reply:
All exported functions and global variable are difficult to change